container-chains/primitives/authorities-noting-inherent/src/mock.rs

```diff
@@ -68,7 +78,11 @@ impl InherentDataProvider for MockAuthoritiesNotingInherentDataProvider {
         let assignment = AuthorityAssignmentSproofBuilder::<NimbusId> {
             authority_assignment: AssignedCollators {
                 orchestrator_chain: vec![],
-                container_chains: Default::default(),
+                container_chains: {
```

Comment:
We should use whatever comes from the `MockAuthoritiesNotingInherentDataProvider`. Additionally we should add own-para-id to `MockAuthoritiesNotingInherentDataProvider`